cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/internal/query/IndexUpdater.java

```diff
@@ -390,4 +319,100 @@ public DBParameter(String tableName, ContentValues contentValues) {
         }
     }
 
+    private static class SequenceNumberForIndexCallable implements SQLCallable<Long> {
```

Comment:
callables should (probably, if we're being consistent with past usage) go into their own files